packages/ra-ui-materialui/src/form/SimpleFormView.tsx

```diff
@@ -49,7 +47,7 @@ const DefaultComponent = ({ children, sx, ...props }) => (
 );
 const DefaultToolbar = <Toolbar />;
 
-export interface SimpleFormViewProps extends FormRenderProps {
+export interface SimpleFormViewProps extends FormRenderProps, StackProps {
```

Comment:
What happens if you provided the `component` prop? I think you need a generic here that defaults to `StackProps`

Reply:
I think the Component should only override the wrapper (CardContent in that case) and not the Stack, because otherwise it's an entirely different Layout.